### PR TITLE
fix: added linting in the workflow

### PR DESCRIPTION
**Pull request by Min2who**
**GitHub Issue:** #289 

**Summary**:

Added linting of pr in every pr

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

No
[JUSTIFICATION]

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    
+jobs:
+
+  Continuous-Integration:
+
+    name: Performs linting on the application
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the Repository
+        uses: actions/checkout@v3
```
Comment (Relequestual) [inline .github/workflows/lint.yml:15]: Looks like we need to use actions/checkout@v4 (at least). Node.js version bump.

**Review by Relequestual — CHANGES_REQUESTED** [review]
Thanks for this PR. I haven't tested it locally but the action seems to run.
Unclear if it is running against any of the files or just finding no results.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (Min2who) [reply .github/workflows/lint.yml]: done @Relequestual

**Comment by Min2who** [comment]
> Thanks for this PR. I haven't tested it locally but the action seems to run. Unclear if it is running against any of the files or just finding no results.



> Thanks for this PR. I haven't tested it locally but the action seems to run. Unclear if it is running against any of the files or just finding no results.

i have tested this action locally [here](https://github.com/Min2who/website/actions/runs/7830453855/job/21364519378?pr=1)

**Comment by Min2who** [comment]
mistakenly deleted my forked repo 🤦‍♀️. Can you give me an idea how do i recover my repo

cc @Relequestual

**Pull request closed without merging**